fsw/src/mm_dump.c

```diff
@@ -103,9 +103,9 @@ bool MM_PeekMem(const MM_PeekCmd_t *CmdPtr, cpuaddr SrcAddress)
     uint16 WordValue      = 0;
     uint32 DWordValue     = 0;
     int32  PSP_Status     = 0;
-    uint32 BytesProcessed = 0;
+    size_t BytesProcessed = 0;
```

Comment:
Later on line 168, this gets assigned to `MM_AppData.HkPacket.BytesProcessed`. What do you think about changing the prototype of `MM_AppData.HkPacket.BytesProcessed` (BytesProcessed of MM_HkPacket_t in file mm_msg.h) to align with this change?

Reply:
Yes Justin - great addition.
I ran through and checked - that member only ever gets assigned to other `size_t`'s or `uint`'s.